src/ocamlorg_frontend/components/pagination.eml

```diff
@@ -1,36 +1,36 @@
-let link ~base_url ~page ~current =
+let link ~base_url ~page ~current ~query =
   let active_class = "border-2 border-current text-primary-600" in
   let non_active_class = "border border-body-100 text-lighter hover:bg-gray-50 hover:no-underline" in
-  <a href="<%s base_url %>?p=<%i page %>"
+  <a href="<%s base_url %>?p=<%i page %><%s if query = "" then "&q=" else "&q=" ^ query %>"
```

Comment:
Since there is some repetition here, I would suggest factoring out common behavior into a function

`let paginated_href ~base_url ~page ~query = ...`

that you use in each of the three places